Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM alpine:3.12
+FROM alpine:3.15
 
-ARG KUBE_VERSION="v1.16.10"
+ARG KUBE_VERSION="v1.23.4"
 ENV AWS_DEFAULT_REGION us-west-2
-ENV AWSCLI_VERSION 1.19.107
+ENV AWSCLI_VERSION 1.22.73
```

Comment:
Note that we officially moved to v2 2 years ago https://aws.amazon.com/blogs/developer/aws-cli-v2-is-now-generally-available/

Reply:
Hmmm interesting... is v2 not used much?

The awscli release notes look like they're only releasing 1.x point releases, and are releasing them frequently: https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst

And the python package we use also is on 1.x: https://pypi.org/project/awscli/

There is this separate python package `awscliv2` but the repo only has 17 stars https://github.com/vemel/awscliv2

Reply:
You need to look at the v2 branch, https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst

Reply:
Ohh got it cool. v2 doesn't need python so probably we can remove all of the python from this chain of images too. Can I do that in a separate PR to keep these atomic? Changing to v2 will mean changing our installation to this method: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-version.html and testing it. Merging the PR as-is still gets us the Alpine and kubectl upgrades and an incremental awscli one and should be more drop in

Reply:
Yes!